README.md

```diff
@@ -717,13 +717,17 @@ See also a test in our example application: [HelloWorldProcessTest](/micronaut-c
 
 When using Gradle we recommend the [Micronaut Application Plugin](https://github.com/micronaut-projects/micronaut-gradle-plugin/blob/master/README.md#micronaut-application-plugin)'s `dockerBuild` task to create a layered Docker image.
 
-Jetty by default only listens on the "localhost" interface. Therefore, you need to configure it to listen on all interfaces by adding the following to your `build.gradle`:
+<details>
+<summary>Workaround for Micronaut 2.4.x</summary>
```

Comment:
I think it would be helpful to specify in the <summary> what the workaround is for. I know that Jetty is our default, but maybe there are also people using another runtime.

Reply:
thanks - fixed